apis/s3/src/main/java/org/jclouds/s3/options/PutObjectOptions.java

```diff
@@ -104,16 +104,16 @@ public void setHeaderTag(@Named(PROPERTY_HEADER_TAG) String headerTag) {
 
 		protected Builder() {
 
-			this.useServerSideEncryption = DEFAULT_SERVER_SIDE_ENCRYPTION;
-			this.acl = DEFAULT_ACCESS_POLICY;
+			this.useServerSideEncryption = DEFAULTS.usesServerSideEncryption();
+			this.acl = DEFAULTS.getAcl();
```

Comment:
Yeah, this was something of a general code cleanup.  I was just trying to make the builder re-use whatever my static DEFAULTS instance had for these values rather than assign them explicitly.

Honestly, I'd prefer not to have this DEFAULTS instance at all; far better to hide that away in the builder constructor and let everybody use the builder to get whatever is appropriate.  The original code was doing something like what's done here, but they were just less direct about it (and called their default instance NONE, the logic of which still escapes me).  I consolidated everything into the DEFAULTS instance to clarify the logic... but that was always something of an incremental step.